openpgp/integration_tests/test_vectors.go

```diff
@@ -1,22 +1,6 @@
-// Copyright 2011 The Go Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file.
-
-package openpgp
-
-import (
-	"bytes"
-	"fmt"
-	"golang.org/x/crypto/openpgp/armor"
-	"golang.org/x/crypto/openpgp/packet"
-	"io"
-	"io/ioutil"
-	"strings"
-	"testing"
-	"time"
-)
-
-type algorithmSet struct {
+package integrationtests
```

Comment:
I think this file should be in a directory called "testdata", so that `go build` ignores it.

Reply:
Hmm the thing is that if it's in another folder, it cannot belong to the same package. I'm fine with that, but it would be silly to make another package just to hold the test vectors and import it. Also I disagree with the use of global constants, so this should probably be a JSON file in a separate `testdata` folder. What do you think?

Reply:
Hmhm, sounds good :+1:

Reply:
Done. This also should be done in other packages, I'll maybe open an issue for this only.